configure.ac

```diff
@@ -245,17 +245,24 @@ dnl ---
 dnl --- Now that we have all the options, let's check for a valid build
 dnl ---
 
+enable_pkey=no
+
 case $target in
      *s390x*)
+	  enable_pkey=yes
 	  ;;
      *s390*)
-	  CFLAGS="$CFLAGS -m31"
+	  CFLAGS="$CFLAGS -m31 -DNO_PKEY"
 	  ;;
      *ppc64* | *x86_64*)
+	  CFLAGS="$CFLAGS -DNO_PKEY"
 	  ;;
      *ppc* | i*86*)
-	  CFLAGS="$CFLAGS -m32"
+	  CFLAGS="$CFLAGS -m32 -DNO_PKEY"
 	  ;;
+	 *)
```

Comment:
indention seems wrong here. Tabs vs spaces ?

Reply:
Yes, this is already a mixture of tabs and spaces ... now tried to at least add the new lines in the same way. Done.